api/capi/include/nnstreamer.h

```diff
@@ -168,7 +168,8 @@ typedef enum {
  */
 typedef enum _ml_tensor_type_e
 {
-  ML_TENSOR_TYPE_INT32 = 0,      /**< Integer 32bit */
+  ML_TENSOR_TYPE_UNKNOWN = 0,    /**< Unknown type */
```

Comment:
This requires another ACR. (it would be a critical issue)
@again4you Please see this change.

Reply:
@dongju-chae 

In my opinion, -1 for this patch. 
If C# code(i.e. TizenFX) is updated simultaneously, then it's OK.
However, it cause ABI break issue.

If an application compiled with updated header installs on old version platform that uses previous header,
ABI break occurs and it does not work properly.
In that case, platform and application should be re-compiled with an updated header.

Is this one and only solution to this problem?

Reply:
Yes, I also think it's too dangerous. How about just using init-functions without changing `enum`?
Is it really problem on performance when we use such init-functions?